react-backend/shared/controllers/feedback.js

```diff
@@ -1,24 +1,48 @@
-const cbAdmin = require('../models/cb-admin');
+const cbAdmin = require('../models/cb_admin');
 
-module.exports = (req, res, next) => {
-  const pgPool = req.app.get('client:psql');
+module.exports = {
+  get: (req, res, next) => {
```

Comment:
You should use the joi validation middleware. I'd suggest one of the following:

1. instead of a function, `exports.get` and `exports.post` export an array which is the middleware stack for that route. This can then be spread into the call to `router.<METHOD>` e.g. `router.post('/route', othermiddleware, ...stack)`.
2. explicitly namespace the `exports.get` object using `handler`, `validation` (and possibly later on `auth`) keys.

Reply:
Is there an example of this in the codebase at the moment? I'm a little lost

Reply:
https://github.com/TwinePlatform/DataPower/blob/master/react-backend/routes/cbauthentication/cb_register.js